src/api/batch-import-export-api.ts

```diff
@@ -0,0 +1,31 @@
+import {PackageExportTransport} from "../interfaces/package-export-transport";
+import {httpClientV2} from "../services/http-client-service.v2";
+import {FatalError} from "../util/logger";
+
+class BatchImportExportApi {
+    public static readonly INSTANCE = new BatchImportExportApi();
+
+    public async findAllActivePackages(flavors: string[], withDependencies: boolean = false): Promise<PackageExportTransport[]> {
```

Comment:
super-nit: It doesn't seem like these methods need to be async (await is not used)